src/chainstate/burn/db/sortdb.rs

```diff
@@ -2430,6 +2463,148 @@ impl SortitionDB {
         Ok(())
     }
 
+    fn apply_schema_3(tx: &mut SortitionHandleTx) -> Result<(), db_error> {
+        // fill in block_commit_parents table
+        info!("Populating block-commit parent table (this might take a while)");
```

Comment:
Can you add something to this message that indicates this is a one-time migration? e.g., `"Migrating sortition database schema from v2 to v3: populating block-commit parents (this might take a while)"`

Reply:
Fixed